app.js

```diff
@@ -0,0 +1,133 @@
+  },
+  {
+    mdltitle: 'Uber Navigation',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s',
+    imageLink: '/assets/last Snapshoot Portfolio.svg',
+    languages: ['html', 'css', 'javascript'],
+    role: 'Lead developer',
+    company: 'UBER',
+    year: '2018',
+    sourceLink: 'https://github.com/Hisoka37/Mobile-Portfolio',
+    demoLink: 'https://github.com/Hisoka37/Mobile-Portfolio',
+  },
+
+];
+
+const cards = document.querySelector('.card');
+const popUpTitle = document.querySelector('.modaltitle');
+const popUpDescription = document.querySelector('.modaldescreption');
+const popUpPhoto = document.querySelector('.modalpicture');
+const popUpSeeSource = document.querySelector('.seesource');
+const popUpSeeLive = document.querySelector('.seelive');
+const popUpContent = document.querySelector('.modalcontent1');
+const modalpopup = document.querySelector('.modalpopup');
+const popUpCompany = document.querySelector('#companyname');
+const popUpRole = document.querySelector('#role');
+const popUpYear = document.querySelector('#year');
+
+cards.innerHTML = '';
+for (let i = 0; i < projectsWindow.length; i += 1) {
+  let techLanguages = '';
+  projectsWindow[i].languages.forEach((projTech) => {
+    techLanguages += `<li class="language">${projTech}</li>`;
+  });
+
+  cards.innerHTML += `
+  <div class="grid-container">
+    <div class="first-card">
+      <img class="card-img" src="${projectsWindow[i].imageLink}" alt="">
+      <div class="my-project">
+
+        <h2 class="project-name">${projectsWindow[i].mdltitle}</h2>
+        <div class="experience">
+          <span class="tag1">${projectsWindow[i].company}</span>
+          <span class="bullet">&#x2022;</span>
+          <span class="tag">${projectsWindow[i].role}</span>
+          <span class="bullet">&#x2022;</span>
+          <span class="tag">${projectsWindow[i].year}</span>
+
+        </div>
+
+        <div class="descreption">
+          <p> ${projectsWindow[i].description}</p>
+        </div>
+
+        <div class="tech">
+          <ul class=" worktech  modaltech ">
+          ${techLanguages}
+          </ul>
+        </div>
+        <button class="see-project" id ="seeproject-${i}">See Project</button>
+      </div>
+    </div>
+  </div>
+
+  `;
+}
+
+for (let i = 0; projectsWindow.length; i += 1) {
+  document.querySelector(`#seeproject-${i}`).addEventListener('click', () => {
+    let modalTechnology = '';
+
+    projectsWindow[i].languages.forEach((languagess) => {
+      modalTechnology += `
+            <li class="language">${languagess}</li>
+          `;
+    });
+
+    popUpTitle.innerHTML = projectsWindow[i].mdltitle;
+    popUpPhoto.src = projectsWindow[i].imageLink;
+    popUpDescription.innerHTML = projectsWindow[i].description;
+    popUpCompany.innerHTML = projectsWindow[i].company;
```

Comment:
As the previous reviewers requested, please match the font size and font-weight with the Figma template. Check the differences below.
<img width="570" alt="Screenshot 2023-02-02 at 12 41 11 AM" src="https://user-images.githubusercontent.com/25887253/216170761-1a9a68b8-814d-424c-9899-9093f07abe7c.png">
<img width="796" alt="Screenshot 2023-02-02 at 12 42 09 AM" src="https://user-images.githubusercontent.com/25887253/216170765-c66c659b-40ce-4e7e-b512-94c0d9a43294.png">

Reply:
@huseyinbiyik this is my third review request and you refuse it because of font-size? i don't need to stick to the figma design for 100% that's and this is OPTIONAL

Reply:
@huseyinbiyik so you required changes for a 4px difference I spend two days struggling with this task and you refuse it because of 4p, don't do this again to someone else, please. 
You should require changes for something worth it.
Thanks

Reply:
Hello @Hisoka37. Sorry for disappointing you but I am following the rules when I am reviewing. The change was requested twice and I am checking if the requested changes were completed. I can totally understand your struggle because I was also a student before. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Reply:
Hello, @huseyinbiyik I am sorry if my comment was inappropriate or something, it was just because I was tired and angry.
Thank you and wish you good luck

Reply:
Hey @Hisoka37. Totally understandable. You did a great job, indeed and I didn't mention that much. Keep up the great work! Happy that the PR is approved now. 😎

![](https://preview.redd.it/tz8vfrq9lx751.jpg?auto=webp&s=e57d3eab556748718a05cb80c546e57c2f985446)